index.html

```diff
@@ -8,6 +8,96 @@
     <link href="./styles.css" rel="stylesheet" >
 </head>
 <body>
-    Bismillah. 
+   <div id="home-page"> 
+        <div id="top-info">
+            <div id="top-left-info">
+                <div class="left-info">Travel</div>
```

Comment:
here no need for two divs. Use one div and set a maximum width for that to wrap the text to another line

Reply:
here also the same above mentioned thing ... to bring the word "goo" below "travel" in middle position ...